kinds/apis/v1beta1/runtimeconfig_types.go

```diff
@@ -26,6 +26,12 @@ type RuntimeConfigSpec struct {
 	AdminConsole AdminConsoleSpec `json:"adminConsole,omitempty"`
 	// LocalArtifactMirrorPort holds the Local Artifact Mirror configuration.
 	LocalArtifactMirror LocalArtifactMirrorSpec `json:"localArtifactMirror,omitempty"`
+
+	// Proxy holds the proxy configuration.
+	Proxy *ProxySpec `json:"proxy,omitempty"`
+
+	// Private CA holds a path to a trusted private CA certificate PEM file.
+	PrivateCA string `json:"privateCA,omitempty"`
```

Comment:
this will not be compatible with prior versions of the product but i think that is ok since it is a new feature.

you will need to set this via the install command. this will also require you to update kots with the latest kinds which means a kinds release is required. i can help with that when ready.

Reply:
@emosbaugh is it expected that a user might pass multiple CA arguments to the installer?  I see that in https://github.com/replicatedhq/embedded-cluster/blob/mitm-proxy-preflight/cmd/embedded-cluster/install.go#L817 we parse the install command and build a map that might hold multiple CA files - I might need to go back to the collector code to make sure that the collector can import more than one

Reply:
thinking something like https://github.com/replicatedhq/embedded-cluster/pull/1318/files#diff-998cd2ce861bc777f6423c98894319d4fa0137fdefef5a3e0bf6987c4ed6d4b2R621 I can use to get just the paths and then pass the paths as a list

Reply:
it looks like they can pass multiple. i think that makes sense.